Project title in the command line

ohorbatiuk opened the pull request:
Issue #1

ohorbatiuk committed:
Issue #1: Load project title from the configuration file.

chmez commented:
Need to fix the issue when a new tab using the project title from the previous tab.

ohorbatiuk committed:
Merge branch 'master' of github.com:chmez/configs into issue/1-project-title-in-command-line

ohorbatiuk committed:
Merge branch 'master' of github.com:chmez/configs into issue/1-project-title-in-command-line

ohorbatiuk committed:
Merge branch 'master' of github.com:chmez/configs into issue/1-project-title-in-command-line

ohorbatiuk committed:
Merge branch 'master' of github.com:chmez/configs into issue/1-project-title-in-command-line

ohorbatiuk committed:
Merge branch 'master' of github.com:chmez/configs into issue/1-project-title-in-command-line

ohorbatiuk committed:
Merge branch 'master' of github.com:chmez/configs into issue/1-project-title-in-command-line

ohorbatiuk committed:
Merge branch 'master' of github.com:chmez/configs into issue/1-project-title-in-command-line

ohorbatiuk committed:
Merge branch 'master' of github.com:chmez/configs into issue/1-project-title-in-command-line

ohorbatiuk committed:
Issue #1: Merge source branch after adding command for add Composer package

ohorbatiuk committed:
Issue #1: Merge source branch after adding command for reverting some existing commit(s)

ohorbatiuk committed:
Issue #1: Merge source branch after making the following changes:

1. Delete duplicated Git command for pulling changes from the "master" branch

2. Change aliases for commands for working with Git submodules

3. Add commands for working with Git stash

ohorbatiuk committed:
Issue #1: Merge source branch after excluding tablespace information from a dump

ohorbatiuk committed:
Issue #1: Merge source branch after extending a command for executing any command in Docker container by defining the second parameter for connecting to the Bash shell when it is not specified

chmez commented:
Replaced by PR #3
